scripts/performance/perf_load/perf_req_gen_get_auth_rules.py

```diff
@@ -0,0 +1,13 @@
+from indy import ledger
+
+from perf_load.perf_req_gen import RequestGenerator
+
+
+class RGGetAuthRules(RequestGenerator):
```

Comment:
I think it would be better (more close to production use cases) to do GET_AUTH_RULE for adding a new CLAIM_DEF.

Reply:
ok